src/sequences/OEISSequenceTemplate.ts

```diff
@@ -43,7 +43,10 @@ export default class OEISSequenceTemplate extends SequenceCached {
     }
 
     async fillCache(): Promise<void> {
-        const urlPrefix = `http://${import.meta.env.VITE_API_URL}/api/`
+        // import.meta.env is basically your configuration.
+        // It's set up by Vite automatically
+        // See https://vitejs.dev/guide/env-and-mode.html.
+        const urlPrefix = `${import.meta.env.VITE_BACKSCOPE_URL}/api/`
```

Comment:
Should `VITE_BACKSCOPE_URL` be https://numberscope.colorado.edu/api/? (Since this is technically where you would go if you wanted backscope, and eventually https://numberscope.colorado.edu will be frontscope.)